src/components/message-screen.js

```diff
@@ -133,6 +143,7 @@ export class MessageScreen extends LitElement {
     }
     img {
       margin-block: 16px;
+      max-height: 140px;
```

Comment:
We don't have any image bigger than this in the figma, this avoids the "globe" svg in the onboaring panel to be so big to cause a scrollbar.